### PR TITLE
Rename services

### PR DESCRIPTION
**Pull request by jimmykarily**
Adapting to changes from here: https://github.com/epinio/epinio/pull/1287

### CODE REVIEW

**Review by enrichman — APPROVED** [review]

File: src/references/settings.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Location
+
+Epinio's settings files is located by default at `~/.config/epinio/settings.yaml`.
+
+The location can be changed from command to command by specifying a
+different path with the global command line option `--settings-file`.
+
+A more permanent change is possible by setting the environment
+variable `EPINIO_SETTINGS` to the desired path.
```
Comment (nunix) [inline src/references/settings.md:21]:
```suggestion
The environment variable `EPINIO_SETTINGS` also allows you to set
a different path for the Settings file.
```

File: src/references/settings.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Contents
+
+Epinio's settings contains
```
Comment (nunix) [inline src/references/settings.md:25]:
```suggestion
Epinio's settings contains:
```

File: src/references/configurations.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## Overview
+
+Configurations are, in general, external components providing some kind of configuration (sic!) to
+applications, without being a direct part of the applications.
+
+Examples of configurations are
+
+  - databases,
+  - message brokers,
+  - interfaces to other kinds of systems,
```
Comment (nunix) [inline src/references/configurations.md:12]:
```suggestion
  - databases
  - message brokers
  - interfaces to other systems
```

File: src/references/settings.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+  - The name of the namespace currently targeted.
```
Comment (nunix) [inline src/references/settings.md:27]:
```suggestion
  - The name of the `namespace` currently targeted
```

File: src/references/settings.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  - Epinio API user name
+  - Epinio API password
+  - Epinio API urls (regular and websocket)
```
Comment (nunix) [inline src/references/settings.md:30]:
```suggestion
  - Epinio API URLs (regular and websocket)
```

File: src/references/settings.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  - Epinio API certificate
+
+The namespace can be changed by running `epinio target` with the
+name of the desired namespace as its single argument.
+
+User name and password are used by the client to authenticate itself
+when talking to Epinio's API server. The `epinio settings update` command
+saves the initial information to the settings.
```
Comment (nunix) [inline src/references/settings.md:38]:
```suggestion
User name and password are used by the client for the authentication
against the Epinio's API server. The `epinio settings update` command
saves the initial information to the settings.
```

Comment (nunix) [reply src/references/settings.md:38]: the "saves the initial information to the settings" means the `settings` are "what" in this context? the in-memory or file settings?

File: src/references/settings.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+The installation uses a the wildcard domain `omg.howdoi.website` and the
+`epinio-ca` issuer by default.
```
Comment (nunix) [inline src/references/settings.md:41]:
```suggestion
The installation uses a wildcard domain (`omg.howdoi.website`) and the
`epinio-ca` issuer by default.
```

File: src/references/settings.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+`epinio settings update` saves the associated CA
+certificate to the settings so that future invocations of the
+client are able to verify the actual certificate when talking to
+Epinio's API server.
```
Comment (nunix) [inline src/references/settings.md:46]: same question as above: what is `settings` in this context?

File: src/references/settings.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+## Commands
+
+The Epinio command line client currently provides 3 commands
+explicitly targeting the settings. These are:
```
Comment (nunix) [inline src/references/settings.md:51]:
```suggestion
The Epinio command line client currently provides three commands
explicitly targeting the settings:
```

File: src/references/settings.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+  1. `epinio target`
+
+     As noted in the previous section, this command changes
+     the namespace to use with all other commands.
```
Comment (nunix) [inline src/references/settings.md:56]:
```suggestion
     This command sets the `namespace` to be used when running the other commands.
```

File: src/references/settings.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+  2. `epinio settings show`
+
+     This command shows the details of the currently stored
+     settings settings. An exception is made for the certificate
+     field, due to its expected size. The command's output only notes
+     if certificate data is present or not.
```
Comment (nunix) [inline src/references/settings.md:63]:
```suggestion
     This command shows the details of the current settings. 
     An exception is made for the certificate field, due to its size, 
     and only outputs if certificate data is present or not.
```

File: src/references/settings.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+  3. `epinio settings update`
+
+     Epinio allows users to switch between multiple installations (on
+     different clusters) by simply re-targeting the cluster to talk to
+     via changes to the  environment variable `KUBECONFIG`.
```
Comment (nunix) [inline src/references/settings.md:69]:
```suggestion
     Epinio allows you to switch between multiple installations (on
     different clusters) by updating the environment variable `KUBECONFIG`
     to the cluster you need to connect.
```

File: src/references/settings.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+
+     When such is done the credentials, cert data, and urls stored in
+     the settings will not match the newly targeted cluster,
+     except by coincidence.
+
+     To be actually able to talk to the newly targeted installation it
+     is necessary to run this command to refresh the stored
+     credentials, cert data, and urls with information retrieved from
+     the new cluster.
```
Comment (nunix) [inline src/references/settings.md:78]:
```suggestion
     Once the `KUBECONFIG` environment variable is updated, you'll
     need to run this command to refresh the stored credentials,
     certificate data and URLs in order to load the settings for the
     selected cluster.
     
     If you don't run this command, the connectivity to the selected
     cluster will fail, as Epinio will still be using the settings for the
     previous cluster.
```

**Review by nunix — COMMENTED** [review]
Rephrased to active voice and asked couple questions about terminology clarification.

**Pull request merged**